heareval/embeddings/task_embeddings.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, module_name: str, model_path: str = None):
             self.model.to(self.device)
         elif isinstance(self.model, tf.Module):
             self.type = TENSORFLOW
-            raise NotImplementedError("TensorFlow embeddings not supported yet.")
+            self.device = None # Tensorflow automatically manages data transfers to device 
```

Comment:
@jorshi want to make a followup issue for this? Just because I don't understand TF2.0 enough to make a great call about this now. In that issue we could also propose profiling this PR.

Reply:
#179